lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -120,7 +126,7 @@ sub _before_render {
 
   my $format = $c->stash('format') || 'json';
   my $res
-    = $args->{exception} ? EXCEPTION() : !$has_spec ? NOT_FOUND() : $c->openapi->not_implemented;
+    = $args->{exception} ? EXCEPTION() : !$has_spec ? NOT_FOUND() : $c->openapi->not_implemented($args, $path);
```

Comment:
You're passing in $path needlessly here. You derive it in the routine and never parse it from the input params anyway.